### ISSUE TITLE
New quest: Segregated ways for bikes and pedestrians?

### ISSUE THREAD

**Issue by rugk** [westnordost/StreetComplete#527]
This time, this quest is really easy, but also really important.

It's about the key [segregated](https://wiki.openstreetmap.org/wiki/Key:segregated):
> This key is used for combined cycle- and footways. If both have their own lane, tag segregated=yes. If they share one lane, tag segregated=no.

That's easy, is not it? (Especially as you often have 

And why is it important?
> This key has no default value and should be tagged on all shared ways! 

And I do not know about other countries, but at least in Germany it is very common.

So my proposal:
Ask condition: `bicycle=yes/designated AND foot=yes/designated on any way` and, respectively, the cases where one of these conditions is implied by the way type (i.e. `highway=cycleway AND foot=designated` e.g.)
Question:
> **Is this path segregated for bikes and pedestrians?**
>
> ![German sign 241: separated foot- and cycleway](https://wiki.openstreetmap.org/w/images/a/aa/120px-Zeichen_241.svg.png) | ![German sign 240: shared foot- and cycleway](https://wiki.openstreetmap.org/w/images/1/11/120px-Zeichen_240.svg.png)
> --- | ---
>  Yes, segregated  |  No, shared way
>
> more answers -> "This is a cycleway only" | "This is a footway only" | usual note leave option

Alternative question: "**Do bikes and pedestrians have their own "lane" or do they use one shared lane together?**"
For countries that do not use the signs above, you may display a [photo](https://wiki.openstreetmap.org/wiki/Key:segregated#Examples).

**Comment by westnordost** [comment]
Must be limited to countries where such signs exist.

**Comment by westnordost** [comment]
Which would also be a research effort.

**Comment by rugk** [comment]
Off-topic, but: Maybe make a tag for "needs-research"?

**Comment by ENT8R** [comment]
What about ways with this sign? How are those tagged correctly?
![](https://www.arcd.de/fileadmin/_processed_/csm_fahrrad-frei_c1dfaac4e4.png)

**Comment by HolgerJeromin** [comment]
@ENT8R 
This would be `highway=footway`,  (`foot=designated`), `bicycly=yes`, `segregated=no`

**Comment by rugk** [comment]
Indeed, that would also be another "other answer" which possibly has to be included.

**Comment by Etua** [comment]
There are signs of that kind in Poland.

**Comment by matkoniecz** [comment]
> Indeed, that would also be another "other answer" which possibly has to be included.

Why? This quest should ask only whatever there are separate areas for cyclists and pedestrians, and anyway it should not be asked for bicycle=yes ways.

**Comment by rugk** [comment]
Yeah after seeing that I also think it should only be asked of bike=designated and foot=designated and bike=yes and foot=yes. If one item is designed and the other yes, then it is such a special street as mentioned before.

**Comment by rugk** [comment]
We should also add "official" as a rare synonym to designated.

**Comment by hsimpson00** [comment]
Hi, even if there are not these signs like rugk showed, you can see if there are segegated ways or not. For example a line on the road or different surface colours.
Even in germany there are cases where there is only a sign like ENT8R showed, but the ways are segregated. This is made because in this case cyclists are also allowed to drive on the road.
So I think this quest would work worldwide!
Regards!

**Comment by rugk** [comment]
> Even in germany there are cases where there is only a sign like ENT8R showed, but the ways are segregated. 

I think a "not" is missing somewhere in this sentence.
(And BTW, IMHO, that would still count as "segregated".)

> So I think this quest would work worldwide!

Possibly, but we need to verify that in all countries we include, it, at least, is sure that a thing such as a white line on such a way is possible, i.e. that ways can be *somwhoe* segregated.
But looking [at the current tag usage](https://taginfo.openstreetmap.org/tags/segregated=yes#map) I think it indeed applies worldwide.

And as I said in the OP:
> For countries that do not use the signs […], you may display a [photo](https://wiki.openstreetmap.org/wiki/Key:segregated#Examples).

**Comment by hsimpson00** [comment]
> I think a "not" is missing somewhere in this sentence.
No. Some examples:

No sign -> no bicycle=designated, but clearly segregated:
https://www.mapillary.com/map/im/QALC0zpveNEIHsYyVtDwag

Sign: footway + bicycle=yes, but segregated:
https://www.mapillary.com/map/im/52sEX9OxdS99ule9i6-Llg

Same as the one before, but firstly not segregated:
https://www.mapillary.com/map/im/1MZQXRDfs52k8NKiS7KZkg
but if you go a few meters, it becomes clearly segregated:
https://www.mapillary.com/map/im/fSl8uUlDWsDm2qYHpKpUCw

The point is, that in Germany, the white bike on the blue sign means you HAVE to use the cycleway, while otherways you are also allowed to use the road. But this doesn't mean it cannot be segregated.

So as a picture, I would take a photo of a path with a wite line in the middle and on the one side a bicycle and on the other side a pedestrian.
For example this: https://www.nuernberg.de/imperia/md/verkehrsplanung/bilder/vpl/radweg_schuckertstrasse.jpg
Or this: http://www.gruen-as.de/2008/22/falsch.jpg

This would work worldwide.

Regards

**Comment by rugk** [comment]
> No sign -> no bicycle=designated, but clearly segregated:
> https://www.mapillary.com/map/im/QALC0zpveNEIHsYyVtDwag

Yes, of course, that is segregated. So you mean `bicycle=yes and foot=yes/designated` would also have to be included? Because even if not designated the ways may be segregated?

> Sign: footway + bicycle=yes, but segregated:
>https://www.mapillary.com/map/im/52sEX9OxdS99ule9i6-Llg

I see no segregation there (just a usual footway-design where also bikes are allowed). And it would contradict to the sign, so segregated=no here.

> Same as the one before, but firstly not segregated:
>https://www.mapillary.com/map/im/1MZQXRDfs52k8NKiS7KZkg
>but if you go a few meters, it becomes clearly segregated:
>https://www.mapillary.com/map/im/fSl8uUlDWsDm2qYHpKpUCw

Okay, that would need a road split and StreetComplete cannot handle this currently.
(And maybe the :bike: painted at the street is even enough to legally say it is designated? Is it?; But actually we do not have to care about this, as StreetComplete will not touch that tag.)

I agree with the image to show however, although there may be better images or so…
We may, however, still show the street sign in addition or **better idea:** just use the "separated" version as a quest icon. I mean the icon is understable for everyone even if they don't know the sign and if they know, they immediately know what the quest is about. And they are reminded of the sign without implying that the sign is *strictly* needed.

So TL;DR: Quest should also ask for `foot/bicycle=yes` values as a legal designation via street sign or so is not needed for segregation. And pictures should be shown instead of street signs for the same reason.

**Comment by hsimpson00** [comment]
Hi, I was only saying that you don't need an explicite traffic sign to segregate a path. And because of this, it isn't necessary to limit this quest this to countrys, where this sign exists.

And of course, there are countrys which don#t have an option to segregate paths, but I guess these countrys have a general bad bicycle infrastructure so there won't be a lot wrong questions.

On the other hand, if there is a path, which is free for both foot and bicycle, it's always good to know if it's segregated or not, even if there is no option for this in this country.

Regards

**Comment by rugk** [comment]
> And of course, there are countrys which don#t have an option to segregate paths,

You mean, no traffic sign or legal concept?
Because later you say:
> it's always good to know if it's segregated or not, even if there is no option for this in this country.

"no option"? So you mean it is not legally possible or what is your definition of "no option"?

So basically, all this quest is asking about is a white line on the ground when it comes down to this point. And thus, we do not need to care about any legal framework, about whether it is "possible"?

**We only needed a black list if there are countries where such a thing as segregated paths just does not exist.** (not legally, but actually in the real world; maybe the paths are either all split or they are just all one thing for any vehicle on it.)
In this case asking the quest would of course make no sense, it would always be not-segregated.

**Comment by rugk** [comment]
So @westnordost may you remove the "help wanted" as we discussed we should not care for the signs, but for the actual separation.
And IMHO it would be fine to ask this quest worldwide…

**Comment by westnordost** [comment]
I did not follow this thread, perhaps someone should summarize the result of the discussion (if it is over).

**Comment by rugk** [comment]
I already did:
> So TL;DR: Quest should also ask for foot/bicycle=yes values as a legal designation via street sign or so is not needed for segregation. And pictures should be shown instead of street signs for the same reason.

The reason is the sign does not have to be there, even in Germany, for a way to be segregated. So the way also does not have to be official (`designated`) . It only has to be drivable by both foot & bike, then we can ask the question whether it is segregated (i.e. there is a visual distinction between the part for bikes and foot) or whether it is not.
I think this makes the whole thing a lot easier.

Where we can use the street sign or a similar design is for the ques icon. That icon makes sense, IMHO (also if you live in a country where such street signs may not exist).

**Comment by westnordost** [comment]
Hmm, by the way, how to map/tag a situation where there is a sidewalk(-alike thingy) with no signs which is for cyclists and pedestrians alike and mapped not as a separate way?
Imagine i.e. an unpaved path right next to the road where it is clear that even in Germany it would be okay to use this way as a cyclist as well because it is sufficiently non-developed (in hamlets etc). In less developed countries, this might be more of the norm.

Basically, https://github.com/westnordost/StreetComplete/issues/527#issuecomment-324309986 but mapped as a separate path.

**Comment by rugk** [comment]
I think, generally, to specify it, you first have to have it mapped as a separate way. The road alone will never be something for pedestrians…

But when it is mapped, I think that would just be `foot=yes` and `bicycle=yes`. And, as it is not segregated in your example, `segregated=no`.
That's at least how I would map it, but I appreciate better suggestion. (Especially I am not sure about the not-mapping-sidewalks-as-separate-paths.)

**Comment by westnordost** [comment]
Well, there is the sidewalk=* tag to map a sidewalk "on" the road and this is even the preferred way of mapping it for cases where the sidewalk is not really separate from the road (i.e. only separated by curb and/or cycle track), so there must be a way to properly tag it.

On 4 September 2017 15:19:37 CEST, rugk <notifications@github.com> wrote:
>I think, generally, to specify it, you first have to have it mapped as
>a separate way. The road alone will never be something for pedestrians…
>
>But when it is mapped, I think that would just be `foot=yes` and
>`bicycle=yes`. And, as it is not segregated in your example,
>`segregated=no`.
>That's at least how I would map it, but I appreciate better suggestion.
>(Especially I am not sure about the
>not-mapping-sidewalks-as-separate-paths.)
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/westnordost/StreetComplete/issues/527#issuecomment-326961763

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by hsimpson00** [comment]
_sidewalk:right:bicycle=yes_
_sigewalk:right:segregated=yes_
something like this?

**Comment by rugk** [comment]
Ah, interesting. If that is possible…

**Comment by hsimpson00** [comment]
http://wiki.openstreetmap.org/wiki/Key:sidewalk

**Comment by rugk** [comment]
In the German example, [this](https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren#cite_ref-oneway_5-4) is mentioned for segregated ways (without sign), at least:
```
cycleway:right=track
cycleway:right:bicycle=designated
cycleway:right:segregated=yes
```

**Comment by hsimpson00** [comment]
Well I guess there is a wrong information in the Wiki. designated means there is a sign (and in Gernamny this means you have to use the track). But here there is no sign...

If you go down, there is another option:
`sidewalk=right
sidewalk:right:foot=designated
sidewalk:right:bicycle=yes
sidewalk:right:oneway=*`

**Comment by rugk** [comment]
Yeah, thought about this too, but maybe it is enough when the way is implicitly "designated", i.e. from the way you see it is a cycleway or so, even without a street sign.

**Comment by westnordost** [comment]
@hsimpson00 this looks about right, thank you! (need this for the cycleway quest, for when the user encounters that situation)

**Comment by hsimpson00** [comment]
No problem ;)

**Comment by andrewharvey** [comment]
I agree with the top comment by rugk, at at least in Australia the same applies and the quest makes sense.

![selection_736](https://user-images.githubusercontent.com/117278/31225446-e225cca2-aa1d-11e7-9921-79a34ec7dd3a.png)

The top is `highway=cycleway` + `foot=designated` + `bicycle=designated` + **`segregated=yes`**.
The bottom is `highway=cycleway` + `foot=designated` + `bicycle=designated` + **`segregated=no`**.

https://www.mapillary.com/map/im/QxaUyDzqGo2vuB6dULOEhw

**Comment by matkoniecz** [comment]
@westnordost is (in theory at least) OK to make this quest with photos of the situation?

I looked for a good photos for this quest and the best that I found is

https://commons.wikimedia.org/wiki/File:Cycleway_(30963809941).jpg https://commons.wikimedia.org/wiki/File:Rousse_-_walk_and_bike_way.jpg for unsegreated (quite good)

https://commons.wikimedia.org/wiki/File:Network_of_city_bike_paths_leading_to_the_university.jpg https://commons.wikimedia.org/wiki/File:Rotterdam_Fietspad_Westzeedijk.jpg https://commons.wikimedia.org/wiki/File:Hauptstra%C3%9Fenradweg_-_Main_road_cycle_lane_in_Amsterdam_(3).jpg for segregated (quite poor)

I am planning to try taking a suitable picture.

**Comment by westnordost** [comment]
I'd tend to say that this would be best shown with a pictogram, just like for the cycleway quest. Do you not agree?

**Comment by matkoniecz** [comment]
I have no preference in this case, with slight preference toward photos as I can make them.

**Comment by westnordost** [comment]
No problem, I like making pictograms

**Comment by matkoniecz** [comment]
OK, I will make a PR with placeholders.

**Comment by rugk** [comment]
BTW PR is here: https://github.com/westnordost/StreetComplete/pull/1135 (@matkoniecz forgot link that :smile: )
